packages/dds/merge-tree/src/test/client.applyMsg.spec.ts

```diff
@@ -539,56 +546,35 @@ _: Local State
-           | DDDDDD---D    | 3:2:D0@0   | DDDC_DD       |            | DDDDC_DD      |            | DDDDDD---D
-           |               |            |     b         |            |      c        |            |
-           | DDDDDD---D    | 4:2:D0@0   | DDDDC_DD      |            | DDDDC_DD      |            | DDDDDD---D
-           |               |            |      b        |            |      c        |            |
-           | DDDDDD---D    | 5:4:D0@0   | DDDDDDDC_DD   |            | DDDDC_DD      |            | DDDDDD---D
-           |               |            |         b     |            |      c        |            |
-           | DDDDDD---D    | 6:4:D1@6,9 | DDDDDD- -_-D  |            | DDDDC_DD      |            | DDDDDD---D
-           |               |            |          b    |            |      c        |            |
-           | DDDDDD---D    |            | DDDDDD- -_-D  | 5:4:D0@0   | DDDDDDDC_DD   |            | DDDDDD---D
-           |               |            |          b    |            |         c     |            |
-           | DDDDDD---D    |            | DDDDDD- -_-D  | 6:4:D1@6,9 | DDDDDD- -_-D  |            | DDDDDD---D
-           |               |            |          b    |            |          c    |            |
-7:6:B0@6   | DDDDDDb ---D  | 7:6:B0@6   | DDDDDD- -b-D  | 7:6:B0@6   | DDDDDDb- -_-D | 7:6:B0@6   | DDDDDDb ---D
-           |               |            |               |            |           c   |            |
-8:6:C0@6   | DDDDDDcb ---D | 8:6:C0@6   | DDDDDDc- -b-D | 8:6:C0@6   | DDDDDDb- -c-D | 8:6:C0@6   | DDDDDDcb ---D
+op types: 0) insert 1) remove 2) annotate
+op         | client A      | op         | client C
+           |               | L:0:C0@0   | _
+           |               |            | C
+1:0:D0@0   | DD            | 1:0:D0@0   | _DD
+           |               |            | C
+2:0:C0@0   | CDD           | 2:0:C0@0   | CDD
+3:2:D0@0   | DDDCDD        | 3:2:D0@0   | DDDCDD
+4:2:D0@0   | DDDDCDD       | 4:2:D0@0   | DDDDCDD
+5:4:D0@0   | DDDDDDDCDD    |            | DDDDCDD
+6:4:D1@6,9 | DDDDDD---D    |            | DDDDCDD
+           | DDDDDD---D    | L:4:C0@5   | DDDDC_DD
+           |               |            |      c
+           | DDDDDD---D    | 5:4:D0@0   | DDDDDDDC_DD
+           |               |            |         c
+           | DDDDDD---D    | 6:4:D1@6,9 | DDDDDD- -_-D
+           |               |            |          c
+7:6:B0@6   | DDDDDDb ---D  | 7:6:B0@6   | DDDDDDb- -_-D
+           |               |            |           c
+8:6:C0@6   | DDDDDDcb ---D | 8:6:C0@6   | DDDDDDb- -c-D
 Client C does not match client A
 ```
      */
-    it.skip("Concurrent insert into removed segment across block boundary", () => {
+    it("Concurrent insert into removed segment across block boundary", () => {
         const clients = createClientsAtInitialState(
             { initialState: "", options: { mergeTreeUseNewLengthCalculations: true } },
```

Comment:
how does this behave without mergeTreeUseNewLengthCalculations?

Reply:
it passes with and without.